pkg/cmd/kubeconfig.go

```diff
@@ -38,7 +38,8 @@ func kubeconfigCmd(rootFlags *pflag.FlagSet) *cobra.Command {
 			This command takes KubeOne manifest which contains information about hosts. It's possible to source information about
 			hosts from Terraform output, using the '--tfjson' flag.
 		`),
-		Example: `kubeone kubeconfig -m mycluster.yaml -t terraformoutput.json`,
+		Example:       `kubeone kubeconfig -m mycluster.yaml -t terraformoutput.json`,
+		SilenceErrors: true,
```

Comment:
What is this option used for?

Reply:
When it's set to default (by default), cobra will automatically report errors. And then we report error, it's the same error printed out two times.

```
        // inside the cobra
	if err != nil {
		// If found parse to a subcommand and then failed, talk about the subcommand
		if cmd != nil {
			c = cmd
		}
		if !c.SilenceErrors {
			c.PrintErrln("Error:", err.Error())
			c.PrintErrf("Run '%v --help' for usage.\n", c.CommandPath())
		}
		return c, err
	}
```

Reply:
However I want to control the way to output so I had to `SilenceErrors: true`, to skip automatic print.